### DISCUSSION TITLE
Is there a command to clear up old history?

### DISCUSSION THREAD

**Discussion by luebbe** [WeblateOrg/weblate#13324] [Q&A] [answered]
When I go to Insights->History, Weblate shows me close to 40.000 pages of history, most of which are obsolete, since they date back several years.

![grafik](https://github.com/user-attachments/assets/457c0323-d0fd-4574-83af-87dad29816c7)

Is there a houskeeping command that allows me to delete history older than xxx days/weeks/months/years?

**Comment by nijel** [comment, marked as answer]
There is currently no such cleanup command available.

**Reply by luebbe** [reply]
Thanks for the reply. So it was not my missing search fu that yielded no information.
Would be nice to have. Shall I create a feature request from this?

**Reply by nijel** [reply]
It definitely makes sense. I always thought that it will be necessary at some point, but was never pushed by circumstances to define how history purging should behave.
